omod/src/main/java/org/openmrs/module/patientdashboard/web/controller/PrintClinicalSummaryController.java

```diff
@@ -73,7 +73,7 @@ public String firstView(@RequestParam("patientId") Integer patientId,
 		
 		Date birthday = patient.getBirthdate();
 		model.addAttribute("age", PatientUtils.estimateAge(birthday));
-		model.addAttribute("ageCategory", PatientUtils.estimateAgeCategory(birthday));
+		//model.addAttribute("ageCategory", PatientUtils.estimateAgeCategory(birthday));

```

Comment:
Why remove and later comment it out? is it NOT required for your use case

Reply:
I commented this out so that I could work on other areas first before coming back to this one